civis/io/_tables.py

```diff
@@ -618,7 +615,9 @@ def dataframe_to_civis(df, database, table, api_key=None, client=None,
                        diststyle=None, distkey=None,
                        sortkey1=None, sortkey2=None,
                        headers=None, credential_id=None,
-                       polling_interval=None,
+                       primary_keys=None, last_modified_keys=None,
+                       execution="immediate",
```

Comment:
`"immediate"` seems like it should be a constant somewhere

Reply:
It looks like other parameters default to strings in this and other functions in this module, which is why I had opted for this approach. I'm happy to make the change for this parameter; however, a broader refactor is outside the scope of this PR.

As an aside, what I think we really ought to be doing here is leveraging [the `enum` module](https://docs.python.org/3/library/enum.html#functional-api) to create named constants for these values, rather than magic strings. However, that is decidedly a larger refactor and would (I believe) require dropping Python 2 support.

Reply:
sure I'm on board w/ this reasoning